### DISCUSSION TITLE
Are there plan to support voice monitoring on Sinotrack devices?

### DISCUSSION THREAD

**Discussion by abosaad11** [eusonlito/GPS-Tracker#27] [General]
Some Sinotrack devices have a microphone and a voice monitoring feature. I looked at the repo and checked the demo website but IU can find anything related to voice monitoring. Any plans to support it?

**Comment by eusonlito** [comment]
What is "support voice monitoring"? 😅 Listening to a conversation through the microphone of the Sinotrack device is only possible from a cell phone.

**Reply by abosaad11** [reply]
What I mean is there's a microphone with the device, you can listen to that microphone with the Sinotrack service. If that's what you mean too, then just out of technical curiosity why couldn't this feature be supported? I guess if it was possible you would already have it anyway, so what is the limitation is my question?


One last question, can I monitor the Sinotrack device with both their service and my own self hosted GPS-Tracker project?

**Reply by eusonlito** [reply]
Yes, some sinotrack devices include a microphone, but to listen to the microphone you need to make (or receive) a call on a phone, it's not something you can do from a web platform. I think their platform includes some VoIP system or Java platform that can allow that, but it's not a simple thing at all.

No, you can use both services at same time, because you only can configure one server at time on device.

**Reply by abosaad11** [reply]
Thank you.
